client/common/api/request.js

```diff
@@ -16,22 +42,18 @@ Object.defineProperty(client, 'base', {
 });
 
 client.interceptors.request.use(config => {
-  const token = window.localStorage.getItem('LMS_TOKEN');
+  const { token } = client.auth;
   if (token) {
-    config.headers['Authorization'] = `JWT ${token}`;
+    config.headers['Authorization'] = `${authScheme} ${token}`;
   } else if (!token && config.headers['Authorization']) {
```

Comment:
Not related to this PR actually, but please remove the `!token &&` part from the condition statement. 😄

Reply:
Lol 👍